src/partials/layout.html

```diff
@@ -9,7 +9,11 @@
       name="description"
       content="Automate code & data workflows with interactive notebooks"
     />
+    {{#if title}}
+    <title>{{title}}</title>
+    {{else}}
     <title>Livebook.dev</title>
+    {{/if}}
```

Comment:
Would this also work?

```suggestion
    <title>{{#if title}}{{title}}{{else}}Livebook.dev{{/if}}</title>
```

Which one do you prefer?

Reply:
I don't have a strong preference. I'll try your suggestion, and we can keep it like that if it works =)

Reply:
It works!